Week3/script.js

```diff
@@ -0,0 +1,273 @@
+
+// 2.7
+
+favoriteAnimals.splice(3, 1);
+
+// 2.8
+
+console.log(favoriteAnimals);
+
+// 2.9
+
+console.log(favoriteAnimals.indexOf("meerkat"));
+
+// 2.10
+
+console.log("The item you are looking for is at index: 1");
+
+        /* *** More JavaScript *** */
+
+        // 1
+
+function threeArguments(a, b, c) {
+  var sum = a + b + c;
+
+  return sum;
+}
+
+        // 2
+
+function colorCar(car) {
+  return car;
+}
+
+let car = "a red car";
+
+console.log(colorCar(car));
+
+        // 3
+
+const person = {
+  name: "Mehmet Ekinci",
+  gender: "M",
+  adres() {
+    console.log("Koemeershoeve");
+  }
+};
+console.log(person);
+
+        // 4
+
+function vehicleType(color,code) {
+  let type = "";
+  if (code === 1) {
+    type = " car";
+  } if (code === 2) {
+    type = " motorbike";
+  }
+
+  console.log("a "+ color + type)
+
+}
+
+vehicleType("blue", 2);
+
+        // 5
+
+console.log(3 === 3 ? "yes" : "no")
+
+        // 6
+
+
+
+function vehicle(color,code,age) {
+  let type,condition = "";
+
+  if (code === 1) {
+    type = " car";
+  } if (code === 2) {
+    type = " motorbike";
+  } if (age >= 2) {
+    condition = " used";
+  } if ((age >= 0) && (age <= 1)) {
+    condition = " new";
+  }
+  
+  console.log("a " + color + condition + type);
+
+}
+vehicle("blue",1,5);
+
+        // 7
+
+let vehicles = ["motorbike", "caravan", "bike", "car", ]
+
+        // 8
+
+console.log(vehicles[2]);
+
+        // 9
+
+function newVehicle(color,code,age) {
+  let type,condition = "";
+  if (code === 1) {
+    type = " motorbike";
+  } if (code === 2) {
+    type = " caravan";
+  }  if (code === 3) {
+    type = " bike";
+  } if (code === 4) {
+    type = " car";
+  } if (age >=2) {
+    condition = " used";
+  } if ((age >= 0) && (age <=1)) {
+    condition = " new";
+  }
+  
+  console.log("a " + color + condition + type)
+}
+newVehicle("green",3,1);
+
+        // 10
+let advertisement = "Amazing Joe's Garage, we service "
+function reAdvertisement(advertisement, vehicles){
+
+  for( let i = 0; i < vehicles.length ;i++){
+    const lastElement = vehicles[vehicles.length-1];
+    const beforLastElement = vehicles[vehicles.length-2];
+    const currentElement = vehicles[i];
+
+    if (currentElement === lastElement) {
+      advertisement = advertisement + currentElement + ".";
+    } else if (currentElement === beforLastElement) {
+      advertisement = advertisement + currentElement + " and ";
+    } else {
+      advertisement = advertisement + currentElement + ", "
+    }
+
+
+  } 
+  return advertisement;
+}
+console.log(reAdvertisement(advertisement,vehicles));
+
+        // 11
+
+// If you add more vehicle to array, you will be add to advertisement.
+function newFunction(){
+vehicles.push("truck");
+ return console.log(reAdvertisement(advertisement,vehicles))
+}
+newFunction();
+
+
+        // 12
+
+let newObject = {};
+
+        // 13
+
+let hackYourFuture = new Object();
+
+hackYourFuture.teachers = "Philipp, Rob, Unmesh, Bonan, Yash";
```

Comment:
It's good practice to create an `array` object for multiple values, instead of a string with `commas`.
So could you change this into an array:)?